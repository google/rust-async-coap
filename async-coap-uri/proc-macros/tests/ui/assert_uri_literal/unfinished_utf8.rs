use proc_macro_hack::proc_macro_hack;

#[proc_macro_hack]
use async_coap_uri_macros::assert_uri_literal;

fn call_macro() {
    assert_uri_literal!("https://www.example.com/a/%F0%90%40");
}

fn main() {}
