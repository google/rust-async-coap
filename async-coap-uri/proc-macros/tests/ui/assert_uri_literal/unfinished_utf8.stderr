error: unfinished utf8
 --> $DIR/unfinished_utf8.rs:7:25
  |
7 |     assert_uri_literal!("https://www.example.com/a/%F0%90%40");
  |                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
